libs/core/langchain_core/callbacks/manager.py

```diff
@@ -2079,6 +2079,12 @@ def _configure(
         )
         for handler in local_handlers_:
             callback_manager.add_handler(handler, False)
+    if run_tree is not None:
+        if run_tree.tags is not None:
+            callback_manager.add_tags(run_tree.tags, False)
```

Comment:
Booleans are easier to read if invoked by name 

```suggestion
            callback_manager.add_tags(run_tree.tags, inherit=False)
```

Reply:
Shouldn't inherit be True though?